queries.yaml

```diff
@@ -51,7 +51,7 @@ layers:
       - vectordatasource.transform.tags_name_i18n
       - vectordatasource.transform.tags_remove
       - vectordatasource.transform.place_population_int
-      - vectordatasource.transform.calculate_default_place_scalerank
+      - vectordatasource.transform.calculate_default_place_min_zoom
```

Comment:
I think everything is explicate in the YAML now, so this may not be necessary any more?